From 9978ec269b7d3a0f0056503f1a8f80c9e7441b90 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 6 Feb 2020 15:25:08 +0200
Subject: [PATCH 06/14] always display the line return key

  * based on
  https://github.com/LineageOS/android_packages_apps_Messaging/commit/06aadd9e31e7bf2d3054dc58fd36197d973dd0ce

	modified:   res/values-ldrtl/styles.xml
	modified:   res/values/styles.xml
---
 res/values-ldrtl/styles.xml | 2 +-
 res/values/styles.xml       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-ldrtl/styles.xml b/res/values-ldrtl/styles.xml
index bd270c7..abfd16c 100644
--- a/res/values-ldrtl/styles.xml
+++ b/res/values-ldrtl/styles.xml
@@ -38,7 +38,7 @@
         <item name="android:background">@null</item>
         <item name="android:scrollHorizontally">false</item>
         <item name="android:textCursorDrawable">@null</item>
-        <item name="android:inputType">textShortMessage|textAutoCorrect|textCapSentences|textMultiLine</item>
+        <item name="android:inputType">textAutoCorrect|textCapSentences|textMultiLine</item>
     </style>
 
     <style name="ConversationComposeSubjectText" parent="ConversationComposeSendText">
diff --git a/res/values/styles.xml b/res/values/styles.xml
index 48dd92f..d374a1c 100644
--- a/res/values/styles.xml
+++ b/res/values/styles.xml
@@ -133,7 +133,7 @@
         <item name="android:background">@null</item>
         <item name="android:scrollHorizontally">false</item>
         <item name="android:textCursorDrawable">@null</item>
-        <item name="android:inputType">textShortMessage|textAutoCorrect|textCapSentences|textMultiLine</item>
+        <item name="android:inputType">textAutoCorrect|textCapSentences|textMultiLine</item>
     </style>
 
     <style name="ConversationComposeSubjectText" parent="ConversationComposeSendText">
-- 
2.17.1

